File: go/raindrops/raindrops.go
package raindrops

import "strconv"

func Convert(n int) string {
	var result string

	if n == 1 {
		return "1"
	}

	if n %  3 == 0 {
		result = "Pling"
	}

	if n % 5 == 0 {
		result += "Plang"
	}

	if n % 7 == 0 {
		result += "Plong"
	}

	if result != "" {
		return result
	}
	return strconv.Itoa(n)
}

File: python/raindrops/raindrops.py
def convert(n):
    drops = ((3, 'Pling'), (5, 'Plang'), (7, 'Plong'))
    """Converts a number to a string according to the raindrop sounds."""
    speak = [s for f, s in drops if n % f == 0]
    return "".join(speak) if speak else str(n)
